cmd/workspace/apps/run_local.go

```diff
@@ -0,0 +1,228 @@
+package apps
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"os/signal"
+	"strings"
+	"syscall"
+	"time"
+
+	"github.com/databricks/cli/cmd/root"
+	"github.com/databricks/cli/libs/appproxy"
+	"github.com/databricks/cli/libs/apps"
+	"github.com/databricks/cli/libs/auth"
+	"github.com/databricks/cli/libs/cmdctx"
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/databricks-sdk-go"
+	"github.com/spf13/cobra"
+)
+
+const SHUTDOWN_TIMEOUT = 15 * time.Second
```

Comment:
I recall you mentioned this timeout is the same as in the real runtime. Can you add a comment explaining this and where you found the constant, or a link to the docs?